builtin/providers/aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -215,6 +219,14 @@ func getAwsCloudWatchPutMetricAlarmInput(d *schema.ResourceData) cloudwatch.PutM
 		params.Unit = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("statistic"); ok {
+		params.Statistic = aws.String(v.(string))
+	}
+
+	if v, ok := d.GetOk("extended_statistic"); ok {
+		params.ExtendedStatistic = aws.String(v.(string))
+	}
+
```

Comment:
If one or the other has to be set, should probably check for that here, and verify that at least one but not both have been set.